### ISSUE TITLE
department selects must be one file

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/groceristar#208]
# Description/Steps to reproduce

### But they have some differences! So might work for both cases.

![image](https://user-images.githubusercontent.com/1469198/31867929-2b2ecb4e-b7a0-11e7-844c-7fe922691e9a.png)

![image](https://user-images.githubusercontent.com/1469198/31867931-2fe5d8bc-b7a0-11e7-82a7-d18c53526110.png)

# Expected result

# Additional information

**Comment by atherdon** [comment]
postpone for now. we have a blocker issue right now

**Comment by atherdon** [comment]
Right now on master we have 3 selects. so before start - we need to discuss plan on this task.
start to work on it, after #245
